x-pack/plugins/cases/server/saved_object_types/migrations/user_actions/alerts.ts

```diff
@@ -26,9 +26,13 @@ export function removeRuleInformation(
   try {
     const { new_value, action, action_field } = doc.attributes;
 
+    if (!isCreateComment(action, action_field)) {
```

Comment:
I think we should check only for the `action_field` to be a comment and the action to be either `create` or `update`  because someone can update an alert attachment from the API.

Reply:
We discussed it offline and because this fix will be in 8.0.1 we will not cover the `update` case.